<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculatrice</class>
 <widget class="QWidget" name="Calculatrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSpinBox" name="spinBoxNombre1"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxOperation">
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxNombre2"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonEgal">
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelResultat">
     <property name="text">
      <string>12</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
